demisto_sdk/commands/split_yml/extractor.py

```diff
@@ -85,6 +85,13 @@ def extract_to_package_format(self) -> int:
             del yaml_obj['image']
             if 'detaileddescription' in yaml_obj:
                 del yaml_obj['detaileddescription']
+        script_obj['script'] = SingleQuotedScalarString('')
+        code_type = script_obj['type']
+        if code_type == TYPE_PWSH and not yaml_obj.get('fromversion'):
```

Comment:
i would also check for `fromVersion`
some time people add it instead of `fromversion` because they see it in other content entities

Reply:
If they add it our schema validation will catch should catch it. I think it is ok.